lib/game.rb

```diff
@@ -0,0 +1,44 @@
+require "./lib/card.rb"
+require "./lib/deck.rb"
+require "./lib/player.rb"
+require "./lib/turn.rb"
+
+class Game 
+  def initialize(player1, player2)
+    @player1 = player1 
+    @player2 = player2
+  end
+
+  def start 
```

Comment:
There is A LOT happening in this method. I would break out the individual pieces of logic to their own methods so they can be better tested and easier to read.